### DISCUSSION TITLE
Issues with the results (very little optimization in drag with <1% gains and no significant deformations)

### DISCUSSION THREAD

**Discussion by antoineblgipsa** [mdolab/dafoam#668] [General]
Hello everyone.

I have an issue with the results of my simulation. When I launch my case, everything seems to work fine. My simulation converges without any bugs. However, when I open my postprocessing results, there is no significant gain in terms of drag. I have less than 0.5 % most of the time. I am trying to modify the shape of a sphere to reduce the drag produced for training.

I have tried to reduce the "primalMinResTol" to 1e-8 and use "primalMinResTolDiff": 5e3. I have also changed the value of the scale in "DVGeo.addLocalDV("shapey", lower=-10.0, upper=10.0, axis="y", scale=0.01, pointSelect=PS)" in my runScript but it doesn't change anything. I was wondering if someone has already encountered this issue before.

**Comment by friedenhe** [comment]
You need to provide more details, like what is your geometry, flow conditions, design variables, and constraints.
